### PR TITLE
Fix console output formatting for simple YAML user guide

### PR DESCRIPTION
**Pull request by satnam6502**

### CODE REVIEW

File: docs/user-guide/simple-yaml.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ can be code reviewed, producing a more robust, reliable and archival system.
 
 ### Running a container from a pod configuration file
 
-```bash
-cd kubernetes
+```console
+$ cd kubernetes
 kubectl create -f ./pod.yaml
```
Comment (bgrant0607) [inline docs/user-guide/simple-yaml.md]: Missed this line. It's a command, not output.

File: docs/user-guide/simple-yaml.md [hunk 2]
```diff
@@ -80,8 +80,8 @@ To run replicated containers, you need a [Replication Controller](replication-co
 A replication controller is responsible for ensuring that a specific number of pods exist in the
 cluster.
 
-```bash
-cd kubernetes
+```console
+$ cd kubernetes
 kubectl create -f ./replication.yaml
```
Comment (bgrant0607) [inline docs/user-guide/simple-yaml.md]: Missed this line also.

File: docs/user-guide/simple-yaml.md (hunk 1 above)
Comment (satnam6502) [inline docs/user-guide/simple-yaml.md]: Thanks, fixed.

File: docs/user-guide/simple-yaml.md (hunk 2 above)
Comment (satnam6502) [inline docs/user-guide/simple-yaml.md]: Thanks, fixed.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**